test/unit-tests/marginCalculator.test.ts

```diff
@@ -150,41 +150,71 @@ contract('MarginCalculator', () => {
     it('Should return cash value for put as strike price - eth price when strike > eth price', async () => {
       const ethPirce = scaleNum(200)
       await oracle.setExpiryPrice(weth.address, closeExpiry, ethPirce)
-      const cashedValue = await calculator.getExpiredCashValue(put.address)
+      const underlying = await put.underlyingAsset()
+      const strike = await put.strikeAsset()
+      const expiryTimestamp = await put.expiryTimestamp()
+      const strikePrice = await put.strikePrice()
+      const isPut = await put.isPut()
+      const cashedValue = await calculator.getExpiredCashValue(underlying, strike, expiryTimestamp, strikePrice, isPut)
       assert.equal(cashedValue.toString(), scaleNum(50))
```

Comment:
I think we should have an util function to get these batched info

Reply:
Yup, can do it in another tests cleaning PR.